src/types/IProduct.d.ts

```diff
@@ -0,0 +1,9 @@
+export interface IProduct {
+  id: number
```

Comment:
use the types that already have for products

Reply:
Done